Dockerfile

```diff
@@ -8,7 +8,7 @@ ENV RUSTC_BOOTSTRAP=0
 
 COPY . /usr/src/app
 
-RUN cd /usr/src/app ; cargo build --release
+RUN apk -U add perl build-base ; cd /usr/src/app ; cargo build --release --features=vendored-openssl
```

Comment:
Why do we need perl?

Reply:
Needed to build openssl, see rebased commit.